### ISSUE TITLE
Next-10 Serverless Deep Dive

### ISSUE THREAD

**Issue by sheplu** [nodejs/next-10#200]
June / July

**Comment by mhdawson** [comment]
@sheplu this is a great idea.  

How about we target June 22 from 10-12 ET?

**Comment by sheplu** [comment]
Could be June 22 or June 29 I think (I would lean a bit more for 29 on my side but would be available for both)

We should validate the date next time we meet

edit: updating the description

**Comment by mhdawson** [comment]
We agreed on the 22nd and 11-1 ET

We should start to reach out to people:
- Members of WinterCG
- Cloud providers that provide serverless
- Node.js collaborators working on startup (Joyee)
- Those interested in a stripped down Node.js?

**Comment by Ethan-Arrowood** [comment]
Shared with Vercel. I'm happy to be point-person, I anticipate more Vercel folks will attend too.

**Comment by Ethan-Arrowood** [comment]
No, general group is fine. But we need more info like when, what, who should attend, etc. 

And I'll be sure to cross post all relevant info for you if that's easiest!

**Comment by mhdawson** [comment]
@Ethan-Arrowood ack, 

Meeting info (will also add to first post in issue)

**Date and Time:** June 22 from 10-12 ET, 14-16 GMT

**Who should attend:** Anybody with an interest in running Node.js in a serverless environment or ensuring that people who run Node.js in serverless environments have a good experience. Also Node.js core collabotors who are interested in helping to make/keep Node.js a good choice for serverless environments

**Agenda:**
1. Is the current state good enough to ensure future success or are there things the project should be doing to improve them?

1. If the answer to the first question is that significant improvements are needed, document specific things that we work on/prioritize

**Link to join:**  https://zoom.us/j/99950131676

**Comment by mhdawson** [comment]
Sent the following email to TSC to ask for help in raising awareness in main Serverless vendors. Appologies in advance if I've left any serverless vendors out in my list, please add any that should be included.

```
The next-10 team has scheduled the next Deep Dive which is to be on Serverless. See https://github.com/nodejs/next-10/issues/200 for more details.

If TSC members have contacts in any of the serverless vendors (AWSM google, Azure, IBM, Cloudflare, Vercel, Red Hat, etc.) I'm asking for help in making somebody from those teams aware of the session and hopefully attend.

I'll cover letting the Red Hat and IBM Serverless teams know. Ethan is already doing the same for Vercel but we need TSC members help for the others.
```

**Comment by mhdawson** [comment]
In case people did not notice I had to change the link for the meeting to - https://meet.google.com/faw-spsk-btp?hs=122&authuser=0

**Comment by mhdawson** [comment]
We ended up with on @wesleytodd, @lholmquist and myself. Whether is was the time, the last minute link switch, not enough work to pull the right people in or just that there is nothing "paining" people using serverless with Node.js I'm not sure. We can discuss that in the next Next-10 meeting to see if we should do a follow up or assume for now that there is nothing to prioritize on the serverless front.
